src/hubmap_translator.py

```diff
@@ -72,6 +72,7 @@ def __init__(self, indices, app_client_id, app_client_secret, token):
             self.DEFAULT_INDEX_WITHOUT_PREFIX: str = indices['default_index']
             self.INDICES: dict = {'default_index': self.DEFAULT_INDEX_WITHOUT_PREFIX, 'indices': self.indices}
             self.DEFAULT_ENTITY_API_URL = self.INDICES['indices'][self.DEFAULT_INDEX_WITHOUT_PREFIX]['document_source_endpoint'].strip('/')
+            self._ontology_api_dict = ontology_api_dict
```

Comment:
When assigning the ontology-api URL, please also strip out trailing slash to avoid double `//` being composed later in the actual call, which will be treated as Not Found by the API Gateway.

Reply:
Done, adjacent to constant declaration in `hubmap_translator.py`.